misc/dist/ios_xcode/godot_ios.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,7 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+        054F8BE62D38852F00B81423 /* MetalFX.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 054F8BE52D38852F00B81423 /* MetalFX.framework */; settings = {ATTRIBUTES = (Weak, ); }; };
```

Comment:
Note the `Weak` attribute, which allows apps to run on OS versions older than iOS 16, which don't have MetalFX.